src/components/templates/ClaimsList/index.tsx

```diff
@@ -2,7 +2,7 @@ import React, {useEffect, useState} from 'react';
 import DateFormatter from '@pega/react-sdk-components/lib/components/helpers/formatters/Date';
 import Button from '../../../components/BaseComponents/Button/Button';
 import PropTypes from "prop-types";
-import { Utils, GBdate} from '../../helpers/utils';
+import { Utils } from '../../helpers/utils';
```

Comment:
Why is this reverted, we need this
I added GBdate as part of a critical defect fix